docs/source/convex-hull.rst

```diff
@@ -0,0 +1,46 @@
+Convex Hull
+===========
+
+Used to find the smallest convex polygon that envolves a set of points in a 2D plane.
+
+.. note::
+
+    Tested on: URI1982
+
+.. code-block:: cpp
+
```

Comment:
Add the necessary includes to use this method, as well as all the typedefs and macros. Be sure to add the code to the /src/geometry/ and check it to see if it compiles